hathor/indexes/manager.py

```diff
@@ -179,11 +188,18 @@ def _manually_initialize(self, tx_storage: 'TransactionStorage') -> None:
 
         tx_storage._update_caches(block_count, tx_count, latest_timestamp, first_timestamp)
 
-    def add_tx(self, tx: BaseTransaction) -> bool:
+    def add_tx(self, tx: BaseTransaction, *, post_consensus: bool = False) -> bool:
```

Comment:
Why don't we remove this method and let the caller choose between `_add_tx_post_consensus` and `_add_tx_pre_consensus`?

Reply:
I wanted to support something like `both=True`, and possible have that as the default. But I also wanted to minimize changes elsewhere.

Reply:
Ok!